source/Glimpse.Ado/AlternateType/GlimpseDbProviderFactory.cs

```diff
@@ -31,9 +33,27 @@ public override bool CanCreateDataSourceEnumerator
 
         private TProviderFactory InnerFactory { get; set; }
 
+        private static bool ShouldNotWrap()
+        {
```

Comment:
maybe first check against the configured default runtime policy as in `GlimpseConfiguration.GetDefaultRuntimePolicy() == RuntimePolicy.Off`